### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: src/z3c/testing/app.py [hunk 1]
```diff
@@ -11,10 +11,9 @@
...
 #
 ##############################################################################
-"""
-$Id$
-"""
-
+from zope.container.tests.test_icontainer import BaseTestIContainer as BTIC
+from zope.container.tests.test_icontainer import DefaultTestData
+from zope.interface.verify import verifyObject, verifyClass
 import unittest
```
Comment (mgedmin) [inline src/z3c/testing/app.py]: PEP-8 requires `import unittest` to be above 3rd-party library imports, separated by one blank line.

File: src/z3c/testing/app.py [hunk 2]
```diff
@@ -89,20 +86,15 @@ def test_verifyClass(self):
 
     def test_verifyObject(self):
         # object test
-        self.assert_(verifyObject(self.getTestInterface(),
-            self.makeTestObject()))
+        self.assert_(verifyObject(
```
Comment (mgedmin) [inline src/z3c/testing/app.py]: Maybe a good opportunity to switch to a non-deprecated `self.assertTrue()` as well.

File: src/z3c/testing/layer.py [hunk 3]
```diff
@@ -43,7 +40,7 @@ class BufferedDatabaseTestLayer(object):
     __bases__ = ()
     path = None
 
-    def __init__(self, config_file=None, module=__module__,
+    def __init__(self, config_file=None, module=__module__,  # noqa
```
Comment (mgedmin) [inline src/z3c/testing/layer.py:43]: Curious: what warning does this suppress?

**Review by mgedmin — APPROVED** [review]
Mostly LGTM.

File: src/z3c/testing/layer.py (hunk 3 above)
Comment (icemac) [reply src/z3c/testing/layer.py:43]: It suppresses: `__module__ is not defined.` I am not sure if this code ever was used successfully.

Comment (mgedmin) [reply src/z3c/testing/layer.py:43]: Is there a pyflakes bug about this false positive warning?  `__module__` is always defined in class scope:

```
>>> __module__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name '__module__' is not defined
>>> class Foo:
...   print(__module__)
... 
__main__
```

Comment (mgedmin) [reply src/z3c/testing/layer.py:43]: https://github.com/PyCQA/pyflakes/issues/259

File: src/z3c/testing/app.py (hunk 1 above)
Comment (icemac) [reply src/z3c/testing/app.py]: This is the part of PEP-8 I do not like. In my projects I have only one block containing all the import statements alphabetically sorted, so `from` imports are above all other ones. (Interestingly the PEP-8 checker  does not complain about miss ordered imports.)

But as it seems to be consensus to follow this part of PEP-8 in the zope projects, keep nagging me if I forget it again.

File: src/z3c/testing/layer.py (hunk 3 above)
Comment (icemac) [reply src/z3c/testing/layer.py:43]: Ah, okay.

File: src/z3c/testing/app.py (hunk 2 above)
Comment (icemac) [reply src/z3c/testing/app.py]: Good idea.

**Pull request merged**